src/dashboard/src/components/api/views.py

```diff
@@ -782,6 +782,7 @@ def _package_create(request):
         payload.get('accession'),
         path,
         payload.get('metadata_set_id'),
+        payload.get('auto_approve', True)
```

Comment:
With a default value so the parameter does not become mandatory. `True` because we're finding this behaviour a better fit but that may change - remember that this is still in beta.